dot/sync/chain_sync.go

```diff
@@ -48,7 +50,14 @@ func (s chainSyncState) String() string {
 	}
 }
 
-var pendingBlocksLimit = maxResponseSize * 32
+var (
+	pendingBlocksLimit = maxResponseSize * 32
```

Comment:
nit a bit out of scope, but I think this could be a constant

Reply:
I get an `unparam` lint error.  Apparently there's a function `newDisjointBlockSet(limit int)` that only gets called with this variable.

Reply:
Update your golangci-lint? 🤔